### DISCUSSION TITLE
GitHub pages look different locally

### DISCUSSION THREAD

**Discussion by ukoehler** [community/community#22249] [Other Feature Feedback, Questions, & Ideas]
<p>Hi there,</p>
<p>I managed to get GitHub pages working locally with Jekyll. The last missing peace is that the theme (dinky) is only applied partially.</p>
<p>GitHub pages:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/d/8d3768477b62fa87c54a975c5d1f8ef057e5910c.png" data-download-href="/uploads/short-url/k9g22U0giKvHZCubsoetD9M02N6.png?dl=1" title="Screenshot_2020-11-19 Hello World" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/13613987/181069726-35b65f22-8f10-44b1-880f-b85a530832f3.png" alt="Screenshot_2020-11-19 Hello World" data-base62-sha1="k9g22U0giKvHZCubsoetD9M02N6" width="690" height="393" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/d/8d3768477b62fa87c54a975c5d1f8ef057e5910c_2_690x393.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/d/8d3768477b62fa87c54a975c5d1f8ef057e5910c_2_1035x589.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/d/8d3768477b62fa87c54a975c5d1f8ef057e5910c_2_1380x786.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/d/8d3768477b62fa87c54a975c5d1f8ef057e5910c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_2020-11-19 Hello World</span><span class="informations">1527×870 18.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Locally:<br>
(Cannot upload the file, so here are the differences:<br>
Text color in header<br>
Instead of the nice red button, I get the text " This project is maintained by <a href="https://github.com/ukoehler" rel="noopener nofollow ugc">ukoehler</a>"<br>
)<br>
I cannot see any errors in the Firefox web console. Some from the Jekyll output:</p>
<blockquote>
<p>bundle exec jekyll serve --trace<br>
Configuration file: /home/ukoehler/GitHubPage/ukoehler.github.io/_config.yml<br>
Source: /home/ukoehler/GitHubPage/ukoehler.github.io<br>
Destination: /home/ukoehler/GitHubPage/ukoehler.github.io/_site<br>
Incremental build: disabled. Enable with --incremental<br>
Generating…<br>
Jekyll Feed: Generating feed for posts<br>
GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.<br>
done in 1.44 seconds.<br>
Auto-regeneration: enabled for ‘/home/ukoehler/GitHubPage/ukoehler.github.io’<br>
Server address: <a href="http://127.0.0.1:4000" rel="noopener nofollow ugc">http://127.0.0.1:4000</a><br>
Server running… press ctrl-c to stop.<br>
[2020-11-19 20:54:09] ERROR `/favicon.ico’ not found.</p>
</blockquote>
<p>Any help is much appreciated</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/ukoehler">@ukoehler</a>! From what I can see from <a href="https://github.com/ukoehler/ukoehler.github.io/">your repository</a> it looks like the theme is applying correctly. You’ve made some modifications to the default HTML template that has changed the <code>h1</code> to an <code>h2</code> resulting in the dark text color.</p>
<p>As for the button issue, it looks like this is because of this section of the template:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/ukoehler/ukoehler.github.io/blob/32167eacbd48333d91ced6ede5c42051172419f4/_layouts/default.html#L21-L29" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/ukoehler/ukoehler.github.io/blob/32167eacbd48333d91ced6ede5c42051172419f4/_layouts/default.html#L21-L29" target="_blank" rel="noopener">ukoehler/ukoehler.github.io/blob/32167eacbd48333d91ced6ede5c42051172419f4/_layouts/default.html#L21-L29</a></h4>
<pre class="onebox"><code class="lang-html"><ol class="start lines" start="21" style="counter-reset: li-counter 20 ;">
<li>{% if site.github.is_project_page %}</li>
<li>  &lt;p class="header"&gt;This project is maintained by &lt;a class="header name" href="{{ site.github.owner_url }}"&gt;{{ site.github.owner_name }}&lt;/a&gt;&lt;/p&gt;</li>
<li>{% endif %}</li>
<li>
<li>{% if site.github.is_user_page %}</li>
<li>  &lt;ul&gt;</li>
<li>    &lt;li&gt;&lt;a class="buttons github" href="{{ site.github.owner_url }}"&gt;GitHub Profile&lt;/a&gt;&lt;/li&gt;</li>
<li>  &lt;/ul&gt;</li>
<li>{% endif %}</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>It’s likely that your local project is mistakenly being seen as a GitHub Project page, but it’s actually a User page on GitHub. This is then triggering the wrong template section to be included in your local environment. If you specifically want one over the other then I’d recommend further modifying your template to only keep the specific HTML from inside the if statement you want to use.</p>

**Comment by ukoehler** [comment]
<p>Hi @ <a href="https://github.community/u/thomasshaped">thomasshaped</a>,</p>
<p>many thanks for your reply. The screen shot is actually from the GitHub page not the local one.</p>
<p>I did indeed change the template from h1 to h2 and found the dark color. That is why I wanted to try it locally for faster turnover. With your help I managed to adapt the local version to behave correctly.</p>
<p>Many thanks for your help!</p>
<p>Uwe</p>
